packages/helm-charts/attestation-service/templates/attestation.statefulset.yaml

```diff
@@ -38,10 +49,13 @@ spec:
             [[ $REPLICA_NAME =~ -([0-9]+)$ ]] || exit 1
             RID=${BASH_REMATCH[1]}
             echo -n $RID >/root/.celo/replica_id
-            echo "Generating private key for rid=$RID"
+            echo "Generating keys for rid=$RID"
             celotooljs.sh generate bip32 --mnemonic "$MNEMONIC" --accountType validator --index $RID > /root/.celo/pkey
-            celotooljs.sh generate bip32 --mnemonic "$MNEMONIC" --accountType attestation --index $RID > /root/.celo/attestationKey
+            celotooljs.sh generate bip32 --mnemonic "$MNEMONIC" --accountType attestation --index $RID > /root/.celo/attestationSignerKey
             celotooljs.sh generate account-address --private-key `cat /root/.celo/pkey` > /root/.celo/address
+            celotooljs.sh generate account-address --private-key `cat /root/.celo/attestationSignerKey` > /root/.celo/attestationSignerAddress
+            echo password > /root/.celo/password
```

Comment:
Is having the password be "password" intentional?

Reply:
Yeah kind of. I figure it didn't matter too much since the node isn't actually exposed and if you have access to the file system that any password is going to be in plaintext anyways